MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -401,7 +417,7 @@ public void onStop() {
       mapRenderer.onStop();
     }
 
-    if (isStarted) {
+    if (isStarted && deactivateFileSource) {
```

Comment:
If we opt to no deactivate FileSource and ConnectivityReceiver here, we should do that in `onDestroy`.